octo/octo.go

```diff
@@ -31,7 +31,7 @@ import (
 const (
 	CraneVersion  = "0.8.0"
 	GoVersion     = "1.20"
-	PackVersion   = "0.29.0"
+	PackVersion   = "0.30.0-pre2"
```

Comment:
they released -pre3 yday

Reply:
Nice =) I'll still with pre2 at the mo as I've tested with it, but def worth moving up once ready =)